src/lang.rs

```diff
@@ -51,6 +52,7 @@ lazy_static::lazy_static! {
             ("kz", "Қазақ"),
             ("ua", "Українська"),
             ("fa", "فارسی"),
+            ("ca", "català"),
```

Comment:
Is it not Català?

Reply:
Yes, thank you, starting with capital letter